pkg/cli/cli.go

```diff
@@ -110,8 +110,14 @@ func CreateSolutionUser(ctx context.Context, o *ClientOption) error {
 	u.role = o.getCredential().role
 	u.cert = o.getCredential().cert
 	return u.Run(ctx, o, CreateUserFunc(func(c *ssoadmin.Client) error {
-		// TODO (fanz): By far, cert data is provided from crt file (by --cert flag)
-		//  Will add an option to generate key-pairs in separate PR.
+
+		if _, err := os.Stat(u.cert); os.IsNotExist(err) {
```

Comment:
We should return an error if `u.cert` already exists.

Reply:
I am thinking if we could do this:
1) Pass the cert file in command line, like `--cert /path/to/certificate/files/k8s-vcp.crt`
2) The file `k8s-vcp.crt` will be assigned to `u.cert`, if it exists, will use this certificate, so the generation key-pair will not be invoked. Otherwise, will reuse the name of `k8s-vcp.crt` to generate key and cert in the same directory `/path/to/certificate/files/`.

In other way, if we want to create key-pair every time, so there is no need to pass the cert file in `--cert`, just directly point to the path and id name like `--cert /path/to/certificate/files/k8s-vcp` 

Please let me know your ideas, thanks. @dougm